parsers/rhea/rhea_test.go

```diff
@@ -0,0 +1,58 @@
+package rhea
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/mattn/go-sqlite3"
+	"log"
+	"os"
+	"testing"
+)
+
+var db *sql.DB
+var rhea Rhea
+
+func TestMain(m *testing.M) {
+	// Read the Compressed Rhea XML to bytes
+	rheaBytes, err := ReadRhea("data/rhea.rdf.gz")
```

Comment:
This is a big thing to commit to git.
<img width="381" alt="Screen Shot 2021-05-15 at 6 25 31 PM" src="https://user-images.githubusercontent.com/1220444/118382482-ef5f4c80-b5aa-11eb-827f-cf7fd2da3bdc.png">

I would do a check for the file and then download it if it is missing. 
More generally does the stuff in the `data` file get stored in the build? Would definitely recommend not adding 5 MBs to the production build.

Reply:
This was definitely something I talked about with @TimothyStiles . I should probably decrease the size before commit. Before that, I wanted to add a few more comments to parser.